Packs/EmployeeOffboarding/.pack-ignore

```diff
@@ -0,0 +1,2 @@
+[file:Employee_Offboarding_-_Gather_User_Information.yml]
```

Comment:
add to PR description all the relations we are ignoring

Reply:
Done